blocklist.txt

```diff
@@ -0,0 +1,49 @@
+# remove some assorted additional plugins
+kolibri/plugins/demo_server/*
+
+# remove python2-only stuff
+kolibri/dist/py2only/*
+
+# Remove cextensions
+kolibri/dist/cext*
+
+# remove source maps
+*.js.map
+
+# remove unused translation files from django and other apps
+kolibri/dist/rest_framework/locale/*
+kolibri/dist/django_filters/locale/*
+kolibri/dist/mptt/locale/*
+
+kolibri/dist/django/contrib/admindocs/locale/*
+kolibri/dist/django/contrib/auth/locale/*
+kolibri/dist/django/contrib/sites/locale/*
+kolibri/dist/django/contrib/contenttypes/locale/*
+kolibri/dist/django/contrib/flatpages/locale/*
+kolibri/dist/django/contrib/sessions/locale/*
+kolibri/dist/django/contrib/humanize/locale/*
+kolibri/dist/django/contrib/admin/locale/*
+
+# remove some django components entirely
+kolibri/dist/django/contrib/gis/*
+kolibri/dist/django/contrib/redirects/*
+kolibri/dist/django/conf/app_template/*
+kolibri/dist/django/conf/project_template/*
+kolibri/dist/django/db/backends/postgresql_psycopg2/*
+kolibri/dist/django/db/backends/postgresql/*
+kolibri/dist/django/db/backends/mysql/*
+kolibri/dist/django/db/backends/oracle/*
+kolibri/dist/django/contrib/postgres/*
+
+# remove bigger chunks of django admin (may not want to do this)
+kolibri/dist/django/contrib/admin/static/*
+kolibri/dist/django/contrib/admin/templates/*
+
+# other assorted testing stuff
+*/test/*
+*/tests/*
+kolibri/dist/tzlocal/test_data/*
+
+# remove some unnecessary apps
+kolibri/dist/redis_cache/*
+kolibri/dist/redis/*
```

Comment:
niice. How's the `.apk` size compare to the old process?

Reply:
It's a few MB smaller, it seems. I think that's probably mostly from dropping all the PyEverywhere runtime code.